### ISSUE TITLE
First Note request for "The EPUB Accessibility exemption property"

### ISSUE THREAD

**Issue by iherman** [w3c/transitions#567]
# Document title, URLs, estimated publication date

- title: The EPUB Accessibility exemption property
- URL: https://www.w3.org/TR/2023/NOTE-epub-a11y-exemption-20231003/
- 2023, October 03 

# Abstract

The exemption property allows EPUB creators to indicate that an EPUB publication that does not meet accessibility conformance requirements has an exemption under the applicable jurisdiction's laws.

# Status of the Document

See https://www.w3.org/TR/2023/NOTE-epub-a11y-exemption-20231003/#sotd

# Link to group's decision to request transition

https://w3c.github.io/pm-wg/minutes/2023-09-11.html#r02

# Is it intended to be a W3C Statement?

No

cc: @wareid @shiestyle @dauwhe @TzviyaSiegman @mattgarrish

**Comment by plehegar** [comment]
This is an unusual note.

* Why is it not on the REC track and published as Note? Was it dropped from an other? The SOTD doesn't say anything about the intent.
* The normative references don't list the European Accessibility Act, despite the fact that it's used in the section A.2

**Comment by iherman** [comment]
> This is an unusual note.
> 
> * Why is it not on the REC track and published as Note? Was it dropped from an other? The SOTD doesn't say anything about the intent.

The usage of that metadata item is not _required_, it is only of an informative nature. If it was part of a Rec track document, it would probably be an informative section altogether. And no, it was not dropped from another specification.

(Formally, the WG is not chartered to issue a new Recommendation, b.t.w.)

> * The normative references don't list the European Accessibility Act, despite the fact that it's used in the section A.2

I leave that question to @mattgarrish and @gregoriopellegrino

**Comment by mattgarrish** [comment]
The note is a response to some changes that ONIX made in their recent code list update to allow publishers to indicate if they are claiming an exemption based on the three EAA categories. We want to make sure publishers can express the same information in either format, at least as much as possible. Official exemptions are not reported this way, but by having the property it allows for collection of stats by vendors, etc. in Europe.

It's not part of the EPUB accessibility specification because claiming an exemption to the EAA (or any accessibility standard) isn't in scope. It would kind of defeat the purpose of the specification to promote ways around it, so I don't see this property becoming a part of it in the future, either. Also, as @iherman says, it wasn't dropped from the standard. This property was raised after we published the RECs.

> > * The normative references don't list the European Accessibility Act, despite the fact that it's used in the section A.2
> 
> I leave that question to @mattgarrish and @gregoriopellegrino

Can we cite legislation? Specref seems limited to technical specifications. Is there an example of how the citation should be formatted?

**Comment by iadawn** [comment]
Does this run the risk of allowing bad actors to automatically circumvent platform accessibility requirements?

**Comment by wareid** [comment]
> Does this run the risk of allowing bad actors to automatically circumvent platform accessibility requirements?

In this very specific case, there are currently only three values, all related to specific EAA exemptions. I believe, if I understand EAA correctly, if a content creator submits a book with this exemption and it ends up in a user's hands and they report an issue it's going to be up to the content creator to prove they meet the exemption criteria.

**Comment by plehegar** [comment]
Another question: Did we get APA to weigh in on the technical content in this document in the past? If not, I believe we ought to give them a chance to look at this.

**Comment by plehegar** [comment]
> > I leave that question to @mattgarrish and @gregoriopellegrino
> 
> Can we cite legislation? Specref seems limited to technical specifications. Is there an example of how the citation should be formatted?

Given that the legislation is explicit part of the definitions of the values, I can't imagine how we cannot list it as a reference if we publish this document.

**Comment by ruoxiran** [comment]
> Another question: Did we get APA to weigh in on the technical content in this document in the past? If not, I believe we ought to give them a chance to look at this.

I will bring this to APA to review.

**Comment by mattgarrish** [comment]
> it's going to be up to the content creator to prove they meet the exemption criteria

Right, I haven't heard of enforcement of accessibility through vendor checking of metadata; they simply don't have the time or resources to verify every publication. At most, this metadata would be used for display in a bookstore. In any case, if they lie in the package document then presumably they'd be lying in their ONIX record, and that's the more authoritative source of metadata for trade publishing in Europe.

It's no different a situation than claiming you conform to a standard while not in fact conforming. Anyone can lie, but there will be penalties for doing so. The EAA has its own requirements for documenting exemptions, so publishers will have to provide proof if questioned on a claim. That's beyond the metadata of a publication, though.

**Comment by mattgarrish** [comment]
> I can't imagine how we cannot list it as a reference if we publish this document.

I don't have an issue with adding a citation, but how do we cite it? Is there a preferred format for legislation? Is it title-only with link? Do you add the initial year of publication or skip years because it's amended periodically? Is the publisher "European Union" or "European Parliament", something else, not necessary?

For example, should it look something like this:

> DIRECTIVE (EU) 2019/882 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 17 April 2019 on the accessibility requirements for products and services. European Union. URL: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882

**Comment by iherman** [comment]
> 
> > DIRECTIVE (EU) 2019/882 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 17 April 2019 on the accessibility requirements for products and services. European Union. URL: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882

This looks good to me, pretty much like our own references. Except that I would not put the title into upper case.

**Comment by iherman** [comment]
> > Another question: Did we get APA to weigh in on the technical content in this document in the past? If not, I believe we ought to give them a chance to look at this.
> 
> I will bring this to APA to review.

Thanks, @ruoxiran. With the current load of APA, do you have any idea how long it would take?

**Comment by mattgarrish** [comment]
> Except that I would not put the title into upper case.

That's the way it's formatted in the act itself. The top of the cited page uses sentence case, though, so probably not an issue.

I just wonder about all the other info with the act, like this:

> Document 32019L0882
> Directive (EU) 2019/882 of the European Parliament and of the Council of 17 April 2019 on the accessibility requirements for products and services (Text with EEA relevance)
> PE/81/2018/REV/1
> OJ L 151, 7.6.2019, p. 70–115 (BG, ES, CS, DA, DE, ET, EL, EN, FR, GA, HR, IT, LV, LT, HU, MT, NL, PL, PT, RO, SK, SL, FI, SV)

I assume we don't need to be this thorough...

In any case, I'll update the spec to add a local biblio reference to what I posted above and let you know when there's a new snapshot.

**Comment by ruoxiran** [comment]
> > > Another question: Did we get APA to weigh in on the technical content in this document in the past? If not, I believe we ought to give them a chance to look at this.
> > 
> > 
> > I will bring this to APA to review.
> 
> Thanks, @ruoxiran. With the current load of APA, do you have any idea how long it would take?

Hi Ivan, APA meets every Wednesday, I will add this to the agenda this week, but how long it takes, depends on everyone's opinions during the meeting. I can make sure that this one could be a high priority for us.

**Comment by ruoxiran** [comment]
APA is OK with this document, see our resolution at: https://www.w3.org/2023/09/27-apa-minutes.html#r01

**Comment by plehegar** [comment]
Assuming the reference citation will get fixed before publication, this is approved.

**Comment by iherman** [comment]
> Assuming the reference citation will get fixed before publication, this is approved.

Thanks @plehegar. @mattgarrish, please ping me when the new snapshot is done and I will issue an official publication request.

**Comment by iherman** [comment]
Publication request issued in https://lists.w3.org/Archives/Group/group-pm-wg-chairs/2023Sep/0002.html (the EU directive reference has been added).

**Comment by iherman** [comment]
Published as https://www.w3.org/TR/epub-a11y-exemption/ today. Closing the issue.
